Gemfile

```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.2"
+ruby '3.2.3'
```

Comment:
This is a bit of a blocker for me at the moment.  In my local development flow, I'm using `rvm` on osx, and it wasn't able to switch to this version of ruby cleanly.  I'll probably have to remove rvm and use homebrew to get to this version - and right now I'm prioritizing getting a release out, I have limited time to work and I'd rather spend it getting the app moving.

I'm probably going to sit on this change for now until the app is further along and I can release it.  I definitely think there is value in getting these things wrapped in containers, but I don't want to do it at the expense of getting an MVP out.

Reply:
Question for @skiunke : Why does the Docker container require 3.2.3? If 3.2.2 works for @kphurley outside of Docker, then it should be possible to get 3.2.2 working inside the Docker container. Just my two cents!

Reply:
@kphurley @esabol I pushed a fix reverting to 3.2.2.
The original error message was somehow cryptic but specifying the version more clearly in the Dockerfile fixed the issue.